lti_consumer/lti_consumer.py

```diff
@@ -756,14 +894,71 @@ def lti_launch_handler(self, request, suffix=''):  # pylint: disable=unused-argu
         Returns:
             webob.response: HTML LTI launch form
         """
-        lti_consumer = LtiConsumer(self)
-        lti_parameters = lti_consumer.get_signed_lti_parameters()
+        if self.lti_version == "lti_1p3":
```

Comment:
I'm trying to follow where this handler will be called for 1.3 requests. Is it possible to have a different handler for 1.3, like `lti_launch_handler_1p3`, so that you don't have to do this switch, and keep `lti_launch_handler` for 1.1 requests? It's only a minor criticism.

Reply:
Sure, I can do this. I've just use this switch to avoid changing the student view, but it makes much more sense to change the student view than this one. 
I'll adjust the PR and push a fix for this.

Reply:
@davestgermain Reverted to the old method and created a new handler for the LTI 1.3 launch. 
Does it look better this way?